packages/flutter/lib/src/painting/binding.dart

```diff
@@ -97,6 +97,10 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
   /// unnecessary memory usage for images. Callers that wish to display an image
   /// above its native resolution should prefer scaling the canvas the image is
   /// drawn into.
+  @Deprecated(
```

Comment:
Please add this information to the doc comment as well.

Reply:
Done